eva-lib/src/main/java/uk/ac/ebi/eva/lib/models/Assembly.java

```diff
@@ -37,8 +37,10 @@
     private String taxonomyCode;
     private String taxonomyEvaName;
 
-    
-    public Assembly(String assemblyAccession, String assemblyChain, String assemblyVersion, String assemblyName, String assemblyCode, 
+    Assembly() {
```

Comment:
"meta/species/list" calls `archiveEvaproDbAdaptor.getSpecies` which returns a `List<Assembly>`. the tests try to deserialize them from JSON to Java object. It is not strictly needed for the application, only for the tests.